eth-contracts/scripts/migrate-contracts.js

```diff
@@ -96,28 +95,25 @@ module.exports = async callback => {
 
   // output to Identity Service
   try {
-    outputJsonConfigFile(getDirectoryRoot(AudiusIdentityService) + '/eth-contract-config.json') 
-  }
-  catch (e) {
+    outputJsonConfigFile(getDirectoryRoot(AudiusIdentityService) + '/eth-contract-config.json')
```

Comment:
good stuff, id probably want to path.join this as well but is non-blocking for this PR